fabric-content-registries-v0/build.gradle

```diff
@@ -1,7 +1,11 @@
 archivesBaseName = "fabric-content-registries-v0"
```

Comment:
I am somewhat unsure if this belongs in content-registries. Since shields are items, maybe this would go better in `item-api`?

Reply:
Yeah since this brings extra dependencies i'd recommend other module. Content registry is more like property registry

Reply:
I have moved it to item-api for now but I'll leave this open for discussion.